pulumi/src/cluster/docker.ts

```diff
@@ -0,0 +1,64 @@
+import { hashElement } from "folder-hash"
+import { Dockerhub } from ".."
+import { hasTag, buildAndPushImage } from "../docker"
+import { getBaseHash, getVolumeReaperHJash as getVolumeReaperHash } from "../hasher"
+
+
+export const buildAndPushDockerImages = async (dockerhub: Dockerhub) => {
```

Comment:
thanks for taking the time to separate some of this stuff out to make it more readable/maintainable